io_test.go

```diff
@@ -36,6 +37,22 @@ func TestFromReaderFile(t *testing.T) {
 	checkResults(t, bf)
 }
 
+func TestFromReaderCorruptFile(t *testing.T) {
+	f, err := os.Open("testdata/broken.bloom")
```

Comment:
Where is this file. Is it missing in this PR?

Reply:
Just added it, must have overlapped with your comment :)